File: tickers/src/main/java/br/com/jeffersonbraga/tickers/model/buyticker/BuyTicker.java
package br.com.jeffersonbraga.tickers.model.buyticker;

import javax.persistence.*;
import java.io.Serializable;
import java.util.Date;

@Entity
@Table(name = "BUY_TICKER")
public class BuyTicker implements Serializable {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer ID = null;

    @Column
    private Long valor = null;

    @Column
    private Long quantidade = null;

    @Column
    private Date data = null;

    @Column
    private String ticker = null;

    public Integer getID() {
        return ID;
    }

    public void setID(Integer ID) {
        this.ID = ID;
    }

    public Long getValor() {
        return valor;
    }

    public void setValor(Long valor) {
        this.valor = valor;
    }

    public Long getQuantidade() {
        return quantidade;
    }

    public void setQuantidade(Long quantidade) {
        this.quantidade = quantidade;
    }

    public Date getData() {
        return data;
    }

    public void setData(Date data) {
        this.data = data;
    }

    public String getTicker() {
        return ticker;
    }

    public void setTicker(String ticker) {
        this.ticker = ticker;
    }
}

File: tickers/src/main/java/br/com/jeffersonbraga/tickers/model/buyticker/BuyTicker.kt
package br.com.jeffersonbraga.tickers.model.buyticker

import java.io.Serializable
import java.math.BigDecimal
import java.util.*
import javax.persistence.*

@Entity
@Table(name = "BUY_TICKER")
class BuyTicker : Serializable {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var iD: Int? = null

    @Column
    var valor: Double? = null

    @Column
    var quantidade: Double? = null

    @Column
    var data: Date? = null

    @Column
    var ticker: String? = null
}
